cms/djangoapps/contentstore/views/assets.py

```diff
@@ -46,8 +45,7 @@
 }
 
 
-mimetypes.init()
-all_mimetypes = list(mimetypes.types_map.values()) + ['text/javascript', 'text/php']
+allowed_mimetypes = settings.ALLOWED_FILE_TYPES
```

Comment:
Add the following setting in common.py and set default values. An error will occur if the values are not set through the plugin.